AssertionError: An unexpected context occurred... with file

When I have a file like so:

```
"""









"""


a = None

```

I get:
```
AssertionError: An unexpected context occurred during parsing of /theabovefile.py It seems not all file lines were tokenized for comment checking.
```

@MiWeiss I think I made a small mistake in the reproduction example, please try this:
```
"""









"""


class a:
    pass
```

docstr-coverage version: 2.0.1 on Ubuntu 20.04.2 LTS with the default config flags (e.g. no `.docstr.yaml` or flags passed to the command, on Python 3.8.5:
```
docstr-coverage reproduce.py 

File: "/home/bjorn/reproduce.py"
Traceback (most recent call last):
  File "/usr/local/bin/docstr-coverage", line 8, in <module>
    sys.exit(execute())
  File "/home/bjorn/.local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/bjorn/.local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/bjorn/.local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/bjorn/.local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/docstr_coverage/cli.py", line 314, in execute
    file_results, total_results = get_docstring_coverage(
  File "/usr/local/lib/python3.8/dist-packages/docstr_coverage/coverage.py", line 254, in get_docstring_coverage
    doc_visitor.visit(parse(source_tree))
  File "/usr/lib/python3.8/ast.py", line 363, in visit
    return visitor(node)
  File "/usr/local/lib/python3.8/dist-packages/docstr_coverage/visitor.py", line 29, in visit_Module
    self.generic_visit(node)
  File "/usr/lib/python3.8/ast.py", line 371, in generic_visit
    self.visit(item)
  File "/usr/lib/python3.8/ast.py", line 363, in visit
    return visitor(node)
  File "/usr/local/lib/python3.8/dist-packages/docstr_coverage/visitor.py", line 33, in visit_ClassDef
    self._visit_helper(node)
  File "/usr/local/lib/python3.8/dist-packages/docstr_coverage/visitor.py", line 44, in _visit_helper
    has_doc = self._has_doc_or_excuse(node)
  File "/usr/local/lib/python3.8/dist-packages/docstr_coverage/visitor.py", line 54, in _has_doc_or_excuse
    return self._has_docstring(node=node) or self._has_excuse(node=node)
  File "/usr/local/lib/python3.8/dist-packages/docstr_coverage/visitor.py", line 78, in _has_excuse
    assert node_start < len(self.tokens), (
AssertionError: An unexpected context occurred during parsing of /home/bjorn/reproduce.py It seems not all file lines were tokenized for comment checking. 
```

--- a/docstr_coverage/visitor.py
+++ b/docstr_coverage/visitor.py
@@ -75,10 +75,6 @@ class DocStringCoverageVisitor(NodeVisitor):
         """Iterates through the tokenize tokens above the passed node to evaluate whether a
         doc-missing excuse has been placed (right) above this nodes begin"""
         node_start = node.lineno
-        assert node_start < len(self.tokens), (
-            "An unexpected context occurred during parsing of {} "
-            "It seems not all file lines were tokenized for comment checking."
-        ).format(self.filename)
 
         # Find the index of first token which starts at the same line as the node
         token_index = -1

--- a/tests/individual_samples/long_doc.py
+++ b/tests/individual_samples/long_doc.py
@@ -1,0 +1,23 @@
+"""
+this is a very long docstring
+
+this is a very long docstring
+this is a very long docstring
+this is a very long docstring
+this is a very long docstring
+this is a very long docstring
+this is a very long docstring
+this is a very long docstring
+this is a very long docstring
+"""
+
+
+class A:
+    """This is the first class in the alphabeth."""
+
+    # docstr-coverage:excused `test ignore after long docstrings`
+    def ignored(self):
+        pass
+
+    def missing(self):
+        pass

--- a/tests/test_coverage.py
+++ b/tests/test_coverage.py
@@ -21,6 +21,8 @@ PARTLY_EXCUSED_FILE_PATH = os.path.join(EXCUSED_SAMPLES_DIRECTORY, "partially_ex
 SAMPLES_C_DIRECTORY = os.path.join("tests", "extra_samples")
 PRIVATE_NO_DOCS_PATH = os.path.join(SAMPLES_C_DIRECTORY, "private_undocumented.py")
 
+INDIVIDUAL_SAMPLES_DIR = os.path.join("tests", "individual_samples")
+
 
 def test_should_report_for_an_empty_file():
     result = analyze([EMPTY_FILE_PATH])
@@ -261,3 +263,16 @@ def test_skip_private():
         "empty": False,
     }
     assert total_results == {"missing_count": 1, "needed_count": 2, "coverage": 50.0}
+
+
+def test_long_doc():
+    """Regression test on issue 79
+
+    Multiline docstrings can be a smoke test when checking
+    the tokenize tokens (which is based on line numbers)."""
+    result = analyze([os.path.join(INDIVIDUAL_SAMPLES_DIR, "long_doc.py")])
+    assert result.count_aggregate().coverage() == 75.0
+    assert result.count_aggregate().num_files == 1
+    # 2 + 1 inline ignore
+    assert result.count_aggregate().found == 3
+    assert result.count_aggregate().needed == 4

